### ISSUE TITLE
Link to data.syncthing.net

### ISSUE THREAD

**Issue by Nutomic** [syncthing/old-web#17]
I don't think this is linked anywhere, except some forum threads.

**Comment by calmh** [comment]
Yeah... And in the web UI where it asks about it. Do we want to publicize it? Is it interesting?

**Comment by AudriusButkevicius** [comment]
To me personally it is.

**Comment by calmh** [comment]
Yes. To me too. :) But what does it mean to the general public? It's not a rhetorical question, I'm just wondering if it really is interesting and if so how we should "promote" it.

**Comment by Nutomic** [comment]
It does show potential users that a lot of people are already using syncthing (successfully). Similar to the way [Tribler](http://statistics.tribler.org/) does it.

syncthing/syncthing-android#273 will probably help with this, seeing as there are ~10k active Android installs which mostly have this disabled.

**Comment by calmh** [comment]
Indeed. I should get around to producing some similar graphs too.

**Comment by AudriusButkevicius** [comment]
:+1: 
I was surprised at the nearly 2k increase in users over 2 weeks while I was away.

**Comment by davidbaumann** [comment]
Maybe put a static, once per day updated png version of "Active Users per Day and Version" on the main page, after "Donate" on the main page?

So, what was acutally happening to the number of users, when changing to 0.12?

**Comment by calmh** [comment]
We disabled usage reporting, as the reported data was added to. Not all users saw the prompt to re-enable it, or accepted to do so.

**Comment by lkwg82** [comment]
So we can show "more than XY users"? This is a very good sign of social trust. This could be with a little text explaining the numbers.
